dulwich/porcelain.py

```diff
@@ -143,6 +144,10 @@ class TimezoneFormatError(Error):
     """Raised when the timezone cannot be determined from a given string."""
 
 
+class AbortCheckout(Error):
```

Comment:
CheckoutError perhaps, for consistency with other exception names?

Reply:
[7fca593](https://github.com/jelmer/dulwich/pull/1148/commits/7fca59313dec9a3b2aa64003aeb85e7626726d58)